dashboard/src/components/Browser.vue

```diff
@@ -1,8 +1,6 @@
 <template>
   <div class="db-browser">
-    <input v-model="search" v-on:keyup.enter="filterOptions" />
-    <button class="search" @click="filterOptions">Search</button>
-    <button class="search-reset" @click="resetSearch">Reset Search</button>
+    <input v-model="search" class="search-box" v-on:keyup="filterOptions" />
```

Comment:
any way to make a double click on an item select it vs scrolling to the bottom?

Reply:
also not sure about keyup vs enter/click for searching modules; seems to bog my circa 2012 i5 and browser down

Reply:
Cool, thanks for checking it out. I've reverted this to the enter/click behavior since it seems like the set/clear timeout is not behaving as expected.